src/test/librbd/fsx.cc

```diff
@@ -3203,6 +3504,13 @@ main(int argc, char **argv)
 			prt("rbd-ggate mode enabled\n");
 			ops = &ggate_operations;
 			break;
+#endif
+#if defined(_WIN32)
+    case 'M':
+      prt("rbd-wnbd mode enabled\n");
+      ops = &wnbd_operations;
+      wnbd_disk = true;
```

Comment:
The ``-r`` and ``-w`` flags only control the offset but not the IO size, so we'd still end up with IO sizes that are not multiples of the sector size.

https://github.com/ceph/ceph/blob/53f89ea09b71162e4d494ffa28b1f86104185c84/src/test/librbd/fsx.cc#L2040-L2042

```
ceph_test_librbd_fsx.exe rbd test_librbd_fsx1 -N 500 -M -L -r 4096 -w 4096 -O
rbd-wnbd mode enabled
pwrite(221184, 40168) failed
short write: 0x0 bytes instead of 0x9ce8
LOG DUMP (1 total operations):
1(  1 mod 256): WRITE    0x36000 thru 0x3fce7   (0x9ce8 bytes)
```

Reply:
Makes sense, ``-t 4096`` as well as ``-O`` are currently optional. Let me know if you're ok with keeping the ``wnbd_flag`` (works with or without ``o_direct``, we only need to ensure that the IO size is a multiple of the sector size).

Reply:
> Let me know if you're ok with keeping the `wnbd_disk` flag (works with or without `o_direct`, we only need to ensure that the IO size is a multiple of the sector size).

I am OK with the idea -- we want fsx on Windows to work without `-O`.  The flag itself can go though, see my other comment.